spaceship/lib/spaceship/portal/app.rb

```diff
@@ -74,12 +133,21 @@ def factory(attrs)
           self.new(attrs)
         end
 
+        # <b>DEPRECATED:</b> Use <tt>all_by_platform</tt> instead.
         # @param mac [Bool] Fetches Mac apps if true
         # @return (Array) Returns all apps available for this account
         def all(mac: false)
-          client.apps(mac: mac).map { |app| self.factory(app) }
+          puts '`all` is deprecated. Please use `all_by_platform` instead.'.red
+          all_by_platform(platform: mac ? Spaceship::Portal::App::MAC : Spaceship::Portal::App::IOS)
```

Comment:
Ok. I am a bit nubish on ruby do you mean to make it a local variable on the Spaceship app object?